src/EnergyPlus/ChilledCeilingPanelSimple.cc

```diff
@@ -1015,29 +1015,62 @@ namespace CoolingPanelSimple {
 		
 		RegisterPlantCompDesignFlow( CoolingPanel( CoolingPanelNum ).WaterInletNode, CoolingPanel( CoolingPanelNum ).WaterVolFlowRateMax );
 
+		bool SizeUAErrorFlag = false;
+		SizeCoolingPanelUA( CoolingPanelNum, SizeUAErrorFlag );
+		if ( SizeUAErrorFlag ) ShowFatalError( "SizeCoolingPanelUA: Program terminated for previous conditions." );
+
+	}
+
+	void
+	SizeCoolingPanelUA(
+					 int const CoolingPanelNum,
+					 bool & SizeUAError
```

Comment:
@jasondegraw Any objections to lumping this comment together with some other C++ modifications that have been requested?  That is already the subject for Followup for new cooling panel ZoneHVAC:CoolingPanel:RadiantConvective:Water (GitHub Issue #5823).